.github/workflows/publish.yml

```diff
@@ -9,7 +9,21 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v1
+      - name: check source code
+        uses: actions/checkout@v3
+        
+      - name: next release version
+        id: nextversion
+        uses: jenkins-x-plugins/jx-release-version@v2.6.11
+        with:
+          previous-version: from-file:charts/tekton-dashboard/Chart.yaml
+
+      - name: Update version in Chart.yaml
+        id: updatechart
+        uses: mikefarah/yq@v4.34.1
+        with:
+          cmd: yq eval '.version = "${{ steps.nextversion.outputs.version }}"' -i charts/tekton-dashboard/Chart.yaml
```

Comment:
So this will release a new version, deploy it, but the version within the repo will remain the same

Reply:
@tomhobson  ah okay. I had a feeling it may do, but was curious to see what would happen in the next step. Ill look into a method of pushing them changes on the step

Reply:
@tomhobson  this has been tested with commit step that I removed (shouldn't have) to the workflow. Not upon merge to main, it will commit the changes to chart.yaml